.github/workflows/run_prod.yml

```diff
@@ -21,21 +21,35 @@ jobs:
     - uses: actions/checkout@v3
     - uses: cvmfs-contrib/github-action-cvmfs@v3
     - uses: eic/run-cvmfs-osg-eic-shell@main
+      env:
+          X509_PROXY: ${{ secrets.X509_PROXY }} 
       with:
         platform-release: "jug_xl:nightly"
         run: |
           eic-info
+          # Generate a timestamp
+          timestamp=$(date '+%Y-%m-%d_%H-%M-%S')
```

Comment:
Do we need `:` as part of the rucio policy package pattern? `date --iso-8601=second` is following a better standard than underscore, and includes TZ.

Reply:
then we would need to update the rucio policy again i think

Reply:
schema update, new pypi release, eic-spack update and then container update to push that through. It's not super critical that goes in the current release though. We can make that change in a separate PR since this only effects the CI.